src/subdomains/generic/kyc/services/kyc-admin.service.ts

```diff
@@ -62,6 +62,10 @@ export class KycAdminService {
     }
   }
 
+  async triggerVideoIdentInternal(userData: UserData): Promise<void> {
+    await this.kycService.initiateStep(userData, KycStepName.IDENT, KycStepType.VIDEO);
+  }
```

Comment:
Maybe we should check if there is already a pending video ident?

Reply:
Yeah thought that too, already 👍

Reply:
Ah I see we got that already, don't need to change anything I would say 👍 check `identService`
Maybe it would make sense to add a try&catch and log the error in case 👍 will add that 👍